tests/installation/disable_grub_timeout.pm

```diff
@@ -53,7 +53,7 @@ sub run {
     # Depending on an optional button "release notes" we need to press "tab"
     # to go to the first tab
     send_key 'tab' unless match_has_tag 'inst-bootloader-settings-first_tab_highlighted';
-    send_key_until_needlematch 'inst-bootloader-options-highlighted', 'right';
+    send_key 'alt-r';
```

Comment:
You need to take into account that some architectures where we have different shortcuts

Reply:
Yes, just seen the other PR (which is better).